### ISSUE TITLE
"How to play"-Page

### ISSUE THREAD

**Issue by YilianSource** [YilianSource/codenames#10]
The site should contain a page that explains the rules of the game.

**Comment by tomlin7** [comment]
yes, the website needs a guide page or a popup, which contains all rules of the game.
